components/SearchResultsList.vue

```diff
@@ -104,4 +103,18 @@ onMounted(() => {
 })
 
 const hasResults = computed(() => resultsStore.searchResultsList.length > 0)
+
+function getLocalizedName(
+  names: Array<{ locale: string, firstName: string, lastName: string }>
+) {
+    const currentLocale = localeStore.activeLocale.code
+    const localePrefix = currentLocale.split('-')[0]
+
+    const selectedName
+    = names.find(name => name.locale === currentLocale)
+      || names.find(name => name.locale.startsWith(localePrefix))
```

Comment:
in a future PR: we really shouldn't hack around the locales having a `-` or not. I think this is a good, quick fix you did here, but we should really fix the locales to be more consistent.